ui/imports/shared/popups/ImportCommunityPopup.qml

```diff
@@ -49,85 +60,138 @@ StatusDialog {
         rightButtons: ObjectModel {
             StatusButton {
                 id: importButton
-                enabled: d.isInputValid
-                loading: (d.isPublicKey && !d.communityFound)
-                text: d.isPrivateKey ? qsTr("Make this an Owner Node")
-                                     : qsTr("Import")
+                enabled: (d.isInputValid && (d.isPrivateKey && d.communityFound ? agreeToKeepOnline.checked : true))
+                loading: (enabled && !d.communityFound)
+                text: d.publicKey ? qsTr("Import") :
+                                    qsTr("Make this device the control node for %1").arg(!loading ? d.communityDetails.name : "")
```

Comment:
This will leave the community name empty, resulting in sth like: "Make this device the control node for "; use a different string then when it's empty?

Reply:
basically not, if it's not loading, it will show the community's name, if it's loading the text won't be visible at all, since it's loading